fetools-app/src/pages/UnitConverter.jsx

```diff
@@ -46,86 +58,144 @@ function UnitConverter() {
     const newEm = newTailwindSize / 4;
     setEm(newEm);
     setPixels(newEm * basePixelSize);
+    updateCssSize(newPixels);
   };
 
   // Handler for the cog (settings) icon click. Toggles the edit mode state.
   const handleCogClick = () => {
     setEditMode(!editMode);
```

Comment:
Here we should also set focus on the base size input field so the user can simply start typing.